src/components/text/Subheading.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components/native'
+
+export const Subheading = styled.Text`
+  font-size: ${props => props.theme.fonts.defaultFontSize};
+  font-family: ${props => props.theme.fonts.contentFontBold};
+  font-weight: ${props => props.theme.fonts.defaultFontWeight};
+`
+
+export const SubheadingText = styled(Subheading)`
```

Comment:
if the `text`is the standard color for a subheading it makes def. sense. But then we don't have the same semantic because i never set color to `header`nor `content`. So if we have a defaultColor for heading and content, too then we should set them everywhere default.
For me the `Basics`part in zeplin is missing like `typo`, `colors`

Reply:
I also think having the same semantic is good.

Reply:
Fine for me, was just an idea :)